doc/langref.html.in

```diff
@@ -5681,14 +5681,24 @@ test "@intToPtr for pointer to zero bit type" {
 
       {#header_open|usingnamespace#}
       <p>
-      {#syntax#}usingnamespace{#endsyntax#} is a top level declaration that imports all the public declarations of
+      {#syntax#}usingnamespace{#endsyntax#} is a declaration that imports all the public declarations of
       the operand, which must be a {#link|struct#}, {#link|union#}, or {#link|enum#}, into the current scope:
       </p>
       {#code_begin|test|usingnamespace#}
 usingnamespace @import("std");
 
 test "using std namespace" {
     testing.expect(true);
+
+    // Also inside struct
```

Comment:
In the current documentation, each test is separated.
Here you are conflating two use cases in one test.